src/screens/AddFriendScreen.js

```diff
@@ -45,43 +43,20 @@ const styles = StyleSheet.create({
 const SectionListBasics = () => {
   return (
     <View style={styles.container}>
-        <Text style= {{textAlign: "center", fontSize:25, marginTop: 20}}>Quick Add</Text>
-        <ScrollView
-            horizontal={false}
-            showsHorizontalScrollIndicator={false}
-            contentContainerStyle={styles.stories}
-          >
-            <AddFriendBitmoji />
-          </ScrollView>
+      <Text style={{ textAlign: "center", fontSize: 25, marginTop: 20 }}>
+        Quick Add
+      </Text>
+      <ScrollView
+        horizontal={false}
+        showsHorizontalScrollIndicator={false}
+        contentContainerStyle={styles.stories}
+      >
+        <AddFriendBitmoji />
+      </ScrollView>
 
```

Comment:
Commented out code is an additional section for the "Add Friends" page. A list of contacts with pfps and add button.